cvmfs/loader.cc

```diff
@@ -1233,8 +1233,6 @@ int FuseMain(int argc, char *argv[]) {
   delete fence_reload_;
   delete loader_exports_;
   delete config_files_;
-  delete repository_name_;
-  delete mount_point_;
```

Comment:
I think they should still be deleted but further down, after their last use.

Reply:
You're right.  I was thinking that setting the pointer to NULL would automatically delete it but that's not the case.